pipeline.Makefile

```diff
@@ -97,7 +100,12 @@ schema-create: $(SCHEMA_FILE)
 
 .PHONY: schema-lint
 schema-lint: $(SCHEMA_FILE)
-	$(RUN) linkml-lint $<
+	@mkdir -p $(VALIDATOR_OUTPUT_DIR)
+	@echo "Linting schema $(SCHEMA_FILE)..."
+	@$(RUN) linkml-lint $< > $(SCHEMA_LINT_LOG) 2>&1 && \
```

Comment:
Why pipe stderr to stdout rather than piping stderr (i.e. `2>`) to the log here?

Reply:
I'm used to seeing this construct but it doesn't matter to me either way. Do you have preference?